<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simutool</class>
 <widget class="QToolBox" name="Simutool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>142</width>
    <height>684</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>学员站</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="page_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>142</width>
     <height>576</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="label">
    <string>个人信息管理</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QToolButton" name="button_userinfo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>基本信息</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/gerenxinxi.png</normaloff>:/pic/images/background/gerenxinxi.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="button_record">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>操作记录</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/操作记录.png</normaloff>:/pic/images/background/操作记录.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="button_userscore">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>历史成绩</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/历史成绩.png</normaloff>:/pic/images/background/历史成绩.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QToolButton" name="buttonMydocument">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>我的文档</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/我的文档.png</normaloff>:/pic/images/background/我的文档.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QToolButton" name="button_logout">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>退出登录</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/退出登录.png</normaloff>:/pic/images/background/退出登录.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>142</width>
     <height>576</height>
    </rect>
   </property>
   <attribute name="label">
    <string>培训考核管理</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QToolButton" name="button_task">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>当前任务</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/任务列表.png</normaloff>:/pic/images/background/任务列表.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="button_revert">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>历史操作再现</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/历史操作再现.png</normaloff>:/pic/images/background/历史操作再现.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QToolButton" name="button_localOprate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>运行操作</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/运行操作.png</normaloff>:/pic/images/background/运行操作.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QToolButton" name="button_Exam">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>知识考试</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/知识考试.png</normaloff>:/pic/images/background/知识考试.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="button_openOix">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>监控模拟操作</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/操作画面.png</normaloff>:/pic/images/background/操作画面.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>142</width>
     <height>576</height>
    </rect>
   </property>
   <attribute name="label">
    <string>仿真工具栏</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QToolButton" name="button_simuControl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>仿真进度控制</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/速度选择.png</normaloff>:/pic/images/background/速度选择.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="button_setScene">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>场景加载</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/场景加载.png</normaloff>:/pic/images/background/场景加载.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="button_setFault">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>故障加载</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/故障设置3.png</normaloff>:/pic/images/background/故障设置3.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QToolButton" name="buttondocuments">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>学习文档</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/学习文档.png</normaloff>:/pic/images/background/学习文档.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QToolButton" name="toolButton_13">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>142</width>
     <height>576</height>
    </rect>
   </property>
   <attribute name="label">
    <string>系统工具</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="3" column="0">
     <widget class="QToolButton" name="button_help">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>帮助系统</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/帮助信息.png</normaloff>:/pic/images/background/帮助信息.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QToolButton" name="toolButton_18">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>待扩展</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="button_sendMessage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>即时通讯</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/联系教员.png</normaloff>:/pic/images/background/联系教员.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="button_setDesplay">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>显示设置</string>
      </property>
      <property name="icon">
       <iconset resource="../SimuTrainee.qrc">
        <normaloff>:/pic/images/background/系统设置.png</normaloff>:/pic/images/background/系统设置.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QToolButton" name="toolButton_19">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>待扩展</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../SimuTrainee.qrc"/>
 </resources>
 <connections/>
</ui>
